lib/adf_base.py

```diff
@@ -49,22 +49,22 @@ def __init__(self, debug = False):
             raise TypeError("'debug' must be a boolean type (True or False)")
 
         self.__debug_fname = ''
+        # Get the current date and time
+        current_timestamp = datetime.now()
+        # Format the datetime object to a string without microseconds
+        dt_str = current_timestamp.strftime('%Y-%m-%d %H:%M:%S')
+        ext = f'{str(dt_str).replace(" ","-")}'
+        debug_fname = f"ADF_debug_{ext}.log"
+        self.__debug_fname = debug_fname
```

Comment:
seems like this could be condensed to
`self.__debug_fname = f"ADF_debug_{datetime.now().strftime('%Y-%m-%d-%H:%M:%S')}.log"`